cmd/server/main.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+import (
+	"luis/goapi/pkg/controller"
+)
+
+func main() {
+	controller.Start()
```

Comment:
the controller package should only contain the handlers not the whole composition